plugins/gitlab-catalog-backend/src/GitlabUserProcessor.ts

```diff
@@ -23,7 +23,7 @@ const GITLAB_DEFAULT_HOST = 'https://gitlab.com';
  */
 export class GitlabUserProcessor implements CatalogProcessor {
   private readonly gitlab: InstanceType<typeof Gitlab>;
-  private readonly logger: Logger;
```

Comment:
todo: note this as a breaking change in the release notes

(though it shouldn't cause too much toil as the `LoggerService` is an interface that seems to support the winston logger